lib/error.js

```diff
@@ -783,6 +784,18 @@ AppError.recoveryCodeNotFound = () => {
   })
 }
 
+AppError.backendServiceFailure = (service, operation) => {
+  return new AppError({
+    code: 500,
+    error: 'Internal Server Error',
+    errno: ERRNO.BACKEND_SERVICE_FAILURE,
```

Comment:
I suspect this will need to be added to the content-server's [list of errors](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/auth-errors.js#L29) so that any errors are correctly logged.